### PR TITLE
attribute table - media and url links

### PR DESCRIPTION
**Pull request by nboisteault**

### CODE REVIEW

**Comment by nboisteault** [comment]
Fix #1167

File: lizmap/modules/lizmap/controllers/service.classic.php [hunk 1]
```diff
@@ -1366,17 +1366,13 @@ public function DescribeFeatureType()
...
-                        $jsonData['name'] = $layername;
-                        $types = array();
-                        $elements = $xml->complexType->complexContent->extension->sequence->element;
-                        foreach ($elements as $element) {
-                            $types[(string) $element->attributes()->name] = (string) $element->attributes()->type;
-                        }
-                        $jsonData['types'] = (object) $types;
+                    $jsonData['name'] = $layer->name;
+                    $types = array();
+                    $elements = $xml->complexType->complexContent->extension->sequence->element;
```
Comment (rldhont) [inline lizmap/modules/lizmap/controllers/service.classic.php]: What if you check `$typename == $this->params['typename'].'Type'` ?

The describeFeatureType can retrieve multiple layers. We probably be sure that if we found `$layer` the parameter typename is for only one layer.

**Comment by josemvm** [comment]
hi @nboisteault after your commit the "Display info" (i) doesn't show images in the right panel for the children tables

**Comment by josemvm** [comment]
https://github.com/3liz/lizmap-web-client/pull/1165

**Comment by josemvm** [comment]
sorry @nboisteault and thanks for your bug fixes

**Comment by nboisteault** [comment]
@josemvm so everything is ok ?

**Comment by josemvm** [comment]
yes @nboisteault, i have re-designed my project :-)

**Pull request merged**